configuration/src/main/resources/db/migration/V20200416__init-schema.sql

```diff
@@ -0,0 +1,134 @@
+CREATE TABLE IF NOT EXISTS Position
+(
```

Comment:
I feel we need indexes - don't forget in the future

Reply:
I'll create a separate task for it